prow/plugins/docs-no-retest/docs-no-retest.go

```diff
@@ -40,7 +41,16 @@ var (
 )
 
 func init() {
-	plugins.RegisterPullRequestHandler(pluginName, handlePullRequest, nil)
+	plugins.RegisterPullRequestHandler(pluginName, handlePullRequest, helpProvider)
+}
+
+func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
+	// The {WhoCanUse, Usage, Examples, Config} fields are omitted because this plugin cannot be
+	// manually triggered and is not configurable.
+	return &pluginhelp.PluginHelp{
+			Description: "The docs-no-retest plugin applies the " + labelSkipRetest + " label to pull requests that only touch documentation type files and thus do not need to be retested against the latest master commit before merging.",
```

Comment:
We should probably mention what types of files the plugin considers as documentation.

Reply:
Done.